gems/pending/metadata/linux/LinuxUtils.rb

```diff
@@ -141,6 +141,44 @@ def self.parse_systemctl_list(lines)
       end.compact!
     end
 
+    def self.collect_interface(interfaces, interface)
```

Comment:
some quick comment here about the fact that (seemingly) two interfaces can share the same mac addr.  Which, side note, seems strange to me.